src/styles/main.scss

```diff
@@ -0,0 +1,33 @@
+@import "variables";
+
+@import "mixins";
+
+@import "animations";
+
+.page {
+  height: 100vh;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0;
+
+  &__clock {
+    position: relative;
+    height: $clockHeight;
+    width: $clockWidth;
+
+    border-radius: 50%;
+    background-color: #000;
+
+    &::before {
+      @include arrow(red);
+      animation: rotate 60s infinite linear;
```

Comment:
you could put animation inside mixin, and pass seconds as second argument

Reply:
Agree, I'"ll do it, and also fix arrows!